generate-stackbrew-library.sh

```diff
@@ -100,6 +100,18 @@ for version; do
 		variantParent="$(awk 'toupper($1) == "FROM" { print $2 }' "$dir/Dockerfile")"
 		variantArches="${parentRepoToArches[$variantParent]}"
 
+		if [ "$variant" = 'bookworm' ] && [ "$version" != '5.1' ] && [ "$version" != '6.0' ]; then
+			# https://github.com/docker-library/redmine/pull/393 (6.1+ deps need newer rust on other arches and just isn't worth the trouble)
+			variantArches="$(jq -r -R <<<"$variantArches" '
+					split(" ")
+					| map(select(
+						IN("arm32v5", "arm32v7", "i386", "mips64le", "ppc64le", "riscv64", "s390x", empty)
+						| not
+					))
+					| join(" ")
+			')"
```

Comment:
I love the energy here, but I think we can simplify this to just `variantArches='amd64 arm64v8'`, right? 😅  or if you want to get fancy, filtering the list to _only_ include those two?  (because nothing else will have published builds on rubygems)

```suggestion
			# https://github.com/docker-library/redmine/pull/393 (6.1+ deps need newer rust on other arches and just isn't worth the trouble)
			variantArches="$(jq <<<"$variantArches" --raw-input --raw-output '
					split(" ")
					| map(select(IN("amd64", "arm64v8")))
					| join(" ")
			')"
```

My logic is that instead of excluding architectures we know aren't supported, we should limit to the architectures we know _are_ supported because that's the information we actually have. :sweat_smile: